tests/Tests/Odesk/Phystrix/CommandTest.php

```diff
@@ -187,6 +187,70 @@ public function testRequestLogOff()
         $this->assertEmpty($this->requestLog->getExecutedCommands());
     }
 
+    /**
+     * Ensure command does not break when configured to log, though one hasn't been injected
+     *
+     * @dataProvider configBoolProvider
+     *
+     * @param bool $log_enabled  whether config is set to use request log
+     */
+    public function testRequestLogNotInjected($log_enabled) {
+
+        // Duplicate some of the class setup in order to bypass requestLog generation
+        $command = new CommandMock();
+        $command->setCommandMetricsFactory($this->commandMetricsFactory);
+        $command->setCircuitBreakerFactory($this->circuitBreakerFactory);
+        $command->setRequestCache(new RequestCache());
+
+        $command->setConfig(new \Zend\Config\Config(array(
+            'requestLog' => array(
+                'enabled' => $log_enabled,
```

Comment:
I know it's probably still PSR-2 compliant, but just for the sake of sticking to something - could you use camelCase for naming variables?